src/style.css

```
@@ -0,0 +1,80 @@
+@font-face {
+  font-family: "Avenir";
+  font-weight: 300px;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: "Avenir";
+  font-weight: bold;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
```

```suggestion
@font-face {
  font-family: "Avenir-heavy";
  font-weight: bold;
  src: url(fonts/Avenir-Heavy.ttf);
}
```
rename it and use it when the bar is in focus, not always.